mflowgen/core/run.py

```diff
@@ -115,16 +116,17 @@ def find_construct_path( design, update ):
 
   # save_construct_path
   #
-  # Save the path to the construct script for future use of --update
+  # Save the path to the construct script and graph_kwargs for future use of --update
   #
 
-  def save_construct_path( s, construct_path ):
+  def save_construct_path( s, construct_path, graph_kwargs ):
```

Comment:
Can you make a separate `save_graph_kwargs()`? It is okay if it is similar code as this one.